### DISCUSSION TITLE
Assets v7 migration guide page reloads to docs v8.x page

### DISCUSSION THREAD

**Discussion by kulabros** [pixijs/pixijs#10454] [General]
Hi,

I'm trying to get to the link, that is in v7 migration guide, more detailed information about new Assets.

https://pixijs.io/guides/basics/assets.html

When I try to visit this page, I can see it for few seconds, and the page immediately reloads to https://pixijs.com/8.x/guides

Is this detailed assets migration help accessible somewhere else, or would it be possible to fix this reload?

Thank you very much

**Comment by bigtimebuddy** [comment]
You want this: https://pixijs.com/8.x/guides/components/assets

**Reply by kulabros** [reply]
Aaah, yes, this is what I was looking for, great, thank you very much! I did not realized it is now in the 8.x docs.
